test/display.test.ts

```diff
@@ -29,14 +40,15 @@ describe('display', () => {
       'hello-world',
       'display-one-dep-three-issues-no-errors.txt',
     );
-    expect(actual).toBe(expected);
+    expect(stripAnsi(actual)).toEqual(stripAnsi(expected));
   });
+
   it('should return expected text for one dependency, one issue with fix, no errors when debug true', async () => {
     const { scanResults } = await scan({ path: helloWorldPath });
     const errors: string[] = [];
     const options: Options = { path: '', debug: true };
     const actual = await display(
-      scanResults,
+      usePosixPath(scanResults),
```

Comment:
alternative names:
`toSlash`, `toPosix`, `useSlash`.

Hmm, and the current one (`usePosixPath`) might be good enough. I'm just elaborating on the alternatives.

Reply:
in upath it's called `toUnix`